File: .vscode/Desafio.py
'''
Para ler e escrever dados em Python, utilizamos as seguintes funções:
- input: lê UMA com dado(s) de entrada do usuário;
- print :imprime um texto de Saída (Output) e pulando uma linha.

'''
#Função útil para calculo do imposto (baseado nas aliquotas)
def calcular_imposto(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 0.05
    elif (salario >= 1100.01 and salario <= 2500):
        aliquota = 0.10
    else :
        aliquota = 0.15
    return aliquota * salario

#Lê os valores de Entrada:
valor_salario = float(input())
valor_beneficios = float(input())

#Calcula o imposto através da função "calcular_imposto":
valor_imposto = calcular_imposto(valor_salario)
#Calcula e imprime a Saíde (com 2 casas decimais):
saida = valor_salario - valor_imposto + valor_beneficios
print(f'{saida:.2f}')
        

File: .vscode/Desafio.js
//Desafios JavaScript na DIO tem funções "gets" e "print" acessíveis globalmente:
//'gets" : lê UMA linha com dado(s) de entrada (inputs) do usuário;
// "set" :imprime um texto de Saída (Output) e pulando uma linha.



//lê os valores de Entrada:
const valorSalario = parseFloat(gets());
const valorBeneficios = parseFloat(gets());

//calcula o imposto através da função "calcularImposto":
 const valorImposto = valorImposto(valorSalario);

//calcula e mprime a Saída (com 2 casas decimais):
const saida = valorSalario - valorImposto + valorBeneficios; 
prin(saida.toFixed(2));

//Função útil para o calculo do imposto (baseado nas aliquotas).
function calcularImposto(salario) {
    Let aliquota;
    if (salario >= 0 && salario <= 1100){
        aliquota = 0.05;
    }else if (valorSalario >= 1100.01 && valorSalario <= 2500){
        valorImposto = 0.10F * valorSalario;
    }   else {
        valorImposto = 0.15F valorSalario;
    }    
    return aliquota * salario
}





 